spec/immutable-data-structures.html

```diff
@@ -637,7 +637,9 @@ <h1>Box ( _arg_ )</h1>
         <p>When the `Box` function is called, the following steps are taken:</p>
         <emu-alg>
           1. If NewTarget is not *undefined*, throw a *TypeError* exception.
-          1. Return a new Box value whose [[Value]] is _arg_.
+          1. If Type(_arg_) is not Object, throw a *TypeError* exception.
```

Comment:
IIUC, this also rejects `Box`es in `Box`es. It was already argued at https://github.com/tc39/proposal-record-tuple/issues/238 that `Box(Box(x))` should be allowed and distinguishable from `Box(x)`. An argument that you agreed with and approved.